src/components/Dashboard/TodoList.vue

```diff
@@ -1568,7 +1578,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin) {
       let subtitle: string = null
       if (EnumUtilities.isStatusDraft(header)) {
         if (this.getNameRequest) {
-          subtitle = `NR APPROVED - ${this.expiresText(this.getNameRequest)}`
+          subtitle = `Name Request APPROVED - ${this.expiresText(this.getNameRequest)}`
```

Comment:
Can you should a screenshot of where this is used? Looks like it's doing the same thing as line 288. Why do we have both?

Reply:
Line 288 is for Authorization APPROVED status; this is for Draft status (draft authorization, not submitted yet).
![image](https://github.com/user-attachments/assets/534b8d75-18bf-45dd-9453-1c1457c371e5)

I did both because for the APPROVED status, there is another line in the subtitles, the look is different

Reply:
To reduce confusion, it would be better to have a single way of doing this.

How about removing the subtitle code here and adding it to the draft continuation in template instead? This will make continuation in subtitles different from other tasks in this file, but at least different states will be handled similarily and reduce confusion around this code on line 1581. What do you think?

Reply:
I don't know if my comment, above, makes sense... What I mean is that continuation tasks will handle subtitles differently from other tasks, but by handling those subtitles in one place only (the template), no one else will ask the question of what line 1581 is doing vs the code in the template.

(Gosh, I hope this comment makes sense.)

Reply:
Do you mean create a new file for the Continuation subtitle template? 
Or just remove line 1581, and put it next to line 288, where we can handle the subtitles for DRAFT, APPROVED and CHANGE_REQUESTED in one place? (I prefer this)

Reply:
I, too, prefer the latter.

Even if it's in 3 places in the template, it won't be a mix of template + script logic.

Reply:
Moved it to line 250, the 3 status subtitles are in one place now (line 250 - 301)